### ISSUE TITLE
AvoidEscapedUnicodeCharacters: Tail comment check failed if there's tabs before tail comment.

### ISSUE THREAD

**Issue by ybbpgfjtey** [checkstyle/checkstyle#3005]
```
E:\test>type TestClass.java
public class TestClass {
    private String unitAbbrev = "\u03bcs"; // There's  no tab before slash, expected no warning.
    private String unitAbbrev = "\u03bcs";  // There's one tab before slash, expected no warning.
}
E:\test> type config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="AvoidEscapedUnicodeCharacters">
            <property name="allowByTailComment" value="true"/>
        </module>
    </module>
</module>
E:\test> java -jar checkstyle-6.16.1-all.jar -c config.xml TestClass.java
[ERROR] E:\test\TestClass.java:3: Unicode escape(s) usage should be avoided. [AvoidEscapedUnicodeCharacters]
Audit done.
Checkstyle ends with 1 errors.
```
** Expected: **
No error.

**Comment by romani** [comment]
> checkstyle-6.5-all.jar 

?? we are 6.16.1 already, you are 10 release behind, you miss about 100 bugfixes.
We do not support old versions.

**Comment by ybbpgfjtey** [comment]
Sorry for use old version, but this bug still exists after I changed to 6.16.1.
```
E:\test>type TestClass.java
public class TestClass {
	private String unitAbbrev = "\u03bcs"; // There's  no tab before slash, expected no warning.
	private String unitAbbrev = "\u03bcs";	// There's one tab before slash, expected no warning.
}
E:\test> type config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
		<module name="AvoidEscapedUnicodeCharacters">
            <property name="allowByTailComment" value="true"/>
        </module>
    </module>
</module>
E:\test> java -jar checkstyle-6.16.1-all.jar -c config.xml TestClass.java
[ERROR] E:\test\TestClass.java:3: Unicode escape(s) usage should be avoided. [AvoidEscapedUnicodeCharacters]
Audit done.
Checkstyle ends with 1 errors.
```

**Comment by romani** [comment]
Fix is merged
